src/Tizen.NUI.Components/Controls/Button.cs

```diff
@@ -623,6 +651,28 @@ public Extents TextPadding
             set => SetValue(TextPaddingProperty, value);
         }
 
+        /// <summary>
+        /// The item (text or icon or both) alignment.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public LinearLayout.Alignment ItemAlignment
+        {
+            get => (LinearLayout.Alignment)GetValue(ItemAlignmentProperty);
+            set => SetValue(ItemAlignmentProperty, value);
+        }
+
+        /// <summary>
+        /// The space between icon and text.
+        /// The value is applied when there exist icon and text both.
+        /// The width value is used when the items are arranged horizontally. Otherwise, the height value is used.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Size2D ItemSpacing
```

Comment:
How about changing this name to CellPadding like LinearLayout?

Reply:
In button, I think `Item` is more suitable than `Cell` so I used `Item`
* ItemAlignment
* ItemSpacing